File: poo/aula13.5/Contato.java
public class Contato { 

    private long id;
    private String nome;
    private String telefone;
    private String email;

    public long getId() {
        return id;
    }
    public void setId(long id) {
        this.id = id;
    }

    public String getNome() { 
        return this.nome;
    }
    public void setNome(String valor) { 
        this.nome = valor;
    }

    public String getTelefone() { 
        return this.telefone;
    }
    public void setTelefone(String valor) { 
        this.telefone = valor;
    }

    public String getEmail() { 
        return this.email;
    }
    public void setEmail(String valor) { 
        this.email = valor;
    }

    @Override
    public String toString() { 
        return "(" + this.id + ") " + this.nome + " - " + this.telefone + " - " + this.email;
    }

}

File: pdmi/aula15/agenda-contato-recyclevioew-firebase-binding/app/src/main/java/com/example/agendacontatorecyclevioewfirebasebinding/model/Contato.kt
package com.example.agendacontatorecyclevioewfirebasebinding.model

data class Contato (
    var id : String? = null,
    var nome : String = "",
    var telefone : String = "",
    var email : String = ""
    ) {
    override fun toString() : String {
        return this.nome + "-" + this.telefone
    }
}
